tests/test_psf.py

```diff
@@ -397,6 +444,46 @@ def testPiffDeterminer_withlog(self):
         """Test Piff determiner with chatty logs."""
         self.checkPiffDeterminer(withlog=True)
 
+    def testPiffDeterminer_stampSize26(self):
+        """Test Piff with a psf stampSize of 26."""
+        with self.assertRaises(ValueError):
+            self.checkPiffDeterminer(stampSize=26)
+
+    def testPiffDeterminer_modelSize26(self):
+        """Test Piff with a psf stampSize of 26."""
+        with self.assertRaises(ValueError):
+            self.checkPiffDeterminer(modelSize=26, stampSize=25)
+
+    def testPiffDeterminer_skyCoords(self):
+        """Test Piff sky coords."""
+
+        self.checkPiffDeterminer(useCoordinates='sky')
+
+    @lsst.utils.tests.methodParameters(angle_degrees=[0, 35, 77, 135])
+    def testPiffDeterminer_skyCoords_with_rotation(self, angle_degrees):
+        """Test Piff sky coords with rotation."""
+
+        wcs = make_wcs(angle_degrees=angle_degrees)
+        self.exposure.setWcs(wcs)
+        self.checkPiffDeterminer(useCoordinates='sky', kernelSize=35)
+
+    @unittest.skipUnless(version.parse(galsim.version) >= version.parse("2.5.2"),
+                         reason="Requires GalSim >= 2.5.2",
+                         )
+    def testPiffDeterminer_skyCoords_rot45(self):
+        """Test Piff sky coords."""
+
+        wcs = make_wcs(angle_degrees=45)
+        self.exposure.setWcs(wcs)
+        self.checkPiffDeterminer(useCoordinates='sky', kernelSize=35)
```

Comment:
@timj Strictly speaking, this would get merged without ever being run. Should I include it, or add a note to add this test case in a later ticket?